### DISCUSSION TITLE
protoc-gen-go version

### DISCUSSION THREAD

**Discussion by bretagne-peiqi** [kedacore/keda#2461] [Q&A / Need Help] [answered]
Hi， I need to know protoc and protoc-gen-go version for externalscaler pb.go file generation.

I used version 3.19.2 but generated different file

**Comment by zroubalik** [comment, marked as answer]
CC @ahmelsayed 

We should probably cover this in Makefile, to have reproducible build pinned to some specific version: https://github.com/kedacore/keda/blob/fe44a97acfa4033da348d21c11750752fd658139/Makefile#L134

and reuse this approach to download the correct `protoc-gen-go` binary: https://github.com/kedacore/keda/blob/fe44a97acfa4033da348d21c11750752fd658139/Makefile#L226-L254

@bretagne-peiqi are you up to contribute this?

**Reply by bretagne-peiqi** [reply]
ok, I will take this.

**Reply by zroubalik** [reply]
Thanks!

**Reply by zroubalik** [reply]
@bretagne-peiqi I have created an issue for this: https://github.com/kedacore/keda/issues/2467
